pkg/serverless/daemon/routes.go

```diff
@@ -35,7 +35,7 @@ type Flush struct {
 
 func (f *Flush) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	log.Debug("Hit on the serverless.Flush route.")
-	if len(os.Getenv(LocalTestEnvVar)) > 0 {
+	if os.Getenv(LocalTestEnvVar) == "true" {
```

Comment:
Looks good, but we may want to look through our codebase and see where we're using `DD_LOCAL_TEST` set to `1` instead of `true` and make sure we don't break those.

Reply:
Good thought. I'll add a conditional for that case.